SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc

```diff
@@ -12,6 +12,147 @@ The content for this appendix may be pushed to SDPi 1.x.
 [sdpi_offset=1]
 === Requirements:  From Narratives to Plug-and-Trust Interfaces
 
+For every device or system interface that supports the elements specified in this technical framework, requirements and capabilities are derived from many different standards, types of standards, and standards development organizations (SDO).  Some are technology focused, such as the ISO/IEEE 11073 standards for health device communication, and others are more focused on quality and risk management, such as the ISO/IEC 80001-1 and ISO 14971 standards.  They are all profiled and implemented and tested in a single interface point, and thus need to be integrated in a coherent and cohesive manner.
+
+In order to manage this diversity and wealth of specifications and standards, the following <<hanging_gardens_framework>> graphic was created:
+
+.Hanging Gardens Framework
+
+image::../images/vol1-diagram-hanging-gardens.svg[]
+
+Though a full explanation of the this framework is beyond this discussion footnote:hanging_gardens_framework_article[A more detailed explanation of this model is provided on the https://confluence.hl7.org/x/4ijxB[IHE-HL7 Gemini _Hanging Gardens Framework_ confluence page].  Last accessed 2022.10.04.], some general observations will help understand the value and use of the framework:
+
+. Real-world Requirements are at the top -- use case requirements, detailing clinical system functions ensure that all requirements and capabilities are rooted in the purposes that technology users are advancing, namely to provide safe
+. <inter layer interfaces Requirements -> Capabilities API, Req Def -> Req Usage >
```

Comment:
This looks like preliminary information - maybe it makes sense to provide a todo or an issue that covers provision of a final text here.

Reply:
Could do that and then finalize in #142, though I'd have to rebase that given how long it has been out there!  The integration of use cases into specific profiles + the updating of the use case specifications with the appropriate metadata could happen in other branches.